### PR TITLE
Fix word order

### PR DESCRIPTION
**Pull request by geryogam**

### CODE REVIEW

**Review by Josh-Cena — COMMENTED** [review]
Could you revert the whitespace changes? We really don't want to split hairs over blank lines.

**Comment by geryogam** [comment]
Sure, done.

File: files/en-us/web/javascript/reference/classes/static_initialization_blocks/index.md [hunk 1]
```diff
@@ -93,31 +90,32 @@ This code shows how to access a public static field.
...
   static {
     console.log(this.field);
   }
 }
+// 'static field'
 ```
 
-The [`super.property`](/en-US/docs/Web/JavaScript/Reference/Operators/super) can be used inside a `static` block to reference properties of a super class.
-This includes static properties, as shown below:
+The [`super.<property>`](/en-US/docs/Web/JavaScript/Reference/Operators/super) can be used inside a `static` block to reference static properties of a super class.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/classes/static_initialization_blocks/index.md:101]: This is not valid syntax anymore; `super.property` was very much on purpose.

Comment (geryogam) [reply files/en-us/web/javascript/reference/classes/static_initialization_blocks/index.md]: @Josh-Cena Done.

**Review by Josh-Cena — APPROVED** [review]
Thanks!

**Comment by geryogam** [comment]
Thanks!

**Pull request merged**